### ISSUE TITLE
bash: digital-cinema-tools-setup: No such file or directory

### ISSUE THREAD

**Issue by cinevfx** [wolfgangw/digital_cinema_tools_distribution#54]
Hi Wolfgang !

The Digital Cinema Tools look like an interesting option to check DCPs.

I am new to Linux and installed WSL and Ubuntu under Windows 11.

Following the setup instructions the terminal showed this:

 wget -p https://git.io/digital-cinema-tools-setup && bash digital-cinema-tools-setup
--2024-09-27 20:26:13--  https://git.io/digital-cinema-tools-setup
Resolving git.io (git.io)... 140.82.112.22
Connecting to git.io (git.io)|140.82.112.22|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://raw.github.com/wolfgangw/digital_cinema_tools_distribution/master/digital-cinema-tools-setup [following]
--2024-09-27 20:26:14--  https://raw.github.com/wolfgangw/digital_cinema_tools_distribution/master/digital-cinema-tools-setup
Resolving raw.github.com (raw.github.com)... 185.199.109.133, 185.199.108.133, 185.199.111.133, ...
Connecting to raw.github.com (raw.github.com)|185.199.109.133|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://raw.githubusercontent.com/wolfgangw/digital_cinema_tools_distribution/master/digital-cinema-tools-setup [following]
--2024-09-27 20:26:14--  https://raw.githubusercontent.com/wolfgangw/digital_cinema_tools_distribution/master/digital-cinema-tools-setup
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.111.133, 185.199.110.133, 185.199.108.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 20365 (20K) [text/plain]
Saving to: ‘git.io/digital-cinema-tools-setup’

git.io/digital-cinema-tools-setup     100%[=========================================================================>]  19.89K  --.-KB/s    in 0.001s

2024-09-27 20:26:14 (14.0 MB/s) - ‘git.io/digital-cinema-tools-setup’ saved [20365/20365]

FINISHED --2024-09-27 20:26:14--
Total wall clock time: 1.1s
Downloaded: 1 files, 20K in 0.001s (14.0 MB/s)
bash: digital-cinema-tools-setup: No such file or directory

Any idea what I have to do to install the Digital Cinema Tools properly ?

Thanks for your help !!!

Marcus

**Comment by wolfgangw** [comment]
Hi, thanks for the message.

I fixed the command on [Setup](https://github.com/wolfgangw/digital_cinema_tools_distribution/wiki/Setup) to make it actually work again. No idea why I had kaputted it at some point.

What happened for you was via ```wget -p [...]``` the downloaded setup script ended up in ```git.io/digital-cinema-tools-setup```. The subsequent call ```bash digital-cinema-tools-setup``` is looking for the setup script in the current directory, not in ```git.io```, thus failing to run.

Sorry for the hassle.
